### DISCUSSION TITLE
Firewall URLs to allow azd

### DISCUSSION THREAD

**Discussion by gbecerra1982** [Azure/azure-dev#4028] [General]
I have a client whose internet access is restricted on their servers. They need to know which URLs are essential for the proper functioning of azd. This seems to be a common issue among several clients. Could you provide me with a detailed list of necessary URLs and any additional information that might help them configure their network accordingly?

**Comment by vhvb1989** [comment]
azd need access to:
- management.azure.com
- devcenter.azure.com
- azurecr.io
- azurecontainerapps.io
- core.windows.net

*If using Government cloud, change `core.windows.net`  and `azurecr.io`, for:
- core.usgovcloudapi.net
- azurecr.us

*If using China cloud, change `core.windows.net`  and `azurecr.io`, for:
- core.chinacloudapi.cn
- azurecr.cn
